pl_examples/basic_examples/mnist_examples/image_classifier_2_lite.py

```diff
@@ -0,0 +1,157 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Here are 5 required steps to convert to LightningLite.
+
+1. Subclass LightningLite and override its run method.
```

Comment:
```suggestion
1. Subclass ``LightningLite`` and override it's ``run`` method.
```

Reply:
it's is a contraction for it is
so the grammar is not correct. 
Also, here LightningLite is considered a name, like PyTorch, so it is fine to not specially format it.